tips/TIP-0026/indexer-rest-api.yaml

```diff
@@ -0,0 +1,757 @@
+          name: timelockedBeforeMilestone
+          schema:
+            type: integer
+          example: 1000
+          description: Return outputs that are timelocked before a certain milestone index.
+        - in: query
+          name: timelockedAfterMilestone
+          schema:
+            type: integer
+          example: 1000
+          description: Return outputs that are timelocked after a certain milestone index.
+        - in: query
+          name: hasExpirationCondition
+          schema:
+            type: boolean
+          example: true
+          description: Filters outputs based on the presence of expiration unlock condition.
+        - in: query
+          name: expiresBefore
+          schema:
+            type: integer
+          example: 1643383242
+          description: Return outputs that expire before a certain Unix timestamp.
+        - in: query
+          name: expiresAfter
+          schema:
+            type: integer
+          example: 1643383242
+          description: Return outputs that expire after a certain Unix timestamp.
+        - in: query
+          name: expiresBeforeMilestone
+          schema:
+            type: integer
+          example: 1000
+          description: Return outputs that expire before a certain milestone index.
+        - in: query
+          name: expiresAfterMilestone
+          schema:
+            type: integer
+          example: 1000
+          description: Return outputs that expire after a certain milestone index.
+        - in: query
+          name: expirationReturnAddress
+          schema:
+            type: string
+          example: iota1qrhacyfwlcnzkvzteumekfkrrwks98mpdm37cj4xx3drvmjvnep6xqgyzyx
+          description: Filter outputs based on the presence of a specific return address in the expiration unlock
+            condition.
+        - in: query
+          name: sender
+          schema:
+            type: string
+          example: iota1qrhacyfwlcnzkvzteumekfkrrwks98mpdm37cj4xx3drvmjvnep6xqgyzyx
+          description: Filters outputs based on the presence of validated Sender (bech32 encoded).
+        - in: query
+          name: tag
+          schema:
+            type: string
+          example: 4ec7f23
+          description: Filters outputs based on matching Tag Block.
+        - in: query
+          name: createdBefore
+          schema:
+            type: integer
+          example: 1643383242
+          description: Return outputs that were created before a certain Unix timestamp.
+        - in: query
+          name: createdAfter
+          schema:
+            type: integer
+          example: 1643383242
+          description: Return outputs that were created after a certain Unix timestamp.
+        - in: query
+          name: offset
+          schema:
+            type: string
+          example: 0c78e998f5177834ecb3bae1596d5056af76e487386eecb19727465b4be86a790200
+          description: Starts the search from the offset (outputID).
```

Comment:
Aren't the first 4 bytes the timestamp of the next outputid and then the output id?

Reply:
See the new definition for `cursor`: `Starts the search from the cursor (confirmationMS+outputId.pageSize).`